src/assets/scss/base.scss

```
@@ -0,0 +1,160 @@
+// Archivos importados
+@import './normalize';
+@import './variables';
+@import './mixins';
+@import './fonts';
+
+/* Contenedores principales */
+body {
+  background-color: #fff;
+}
+
+.container {
+  margin-top: 0;
+  @include flex;
+}
+
+/*Texto*/
+h2 {
+  color: $negro;
+}
+
+h3 {
+  color: $negro;
+}
+
+p, button {
+  font-weight: 300;
+}
+
+/* Header */
+.header {
+  @include flex;
+  align-items: center;
+  background-color: $negro;
+  .container {
+    justify-content: space-between;
+  }
+  &__logo {
+    cursor: pointer;
+  }
+  &__login {
+    button {
```

El objetivo de BEM es usar clases en vez de los elementos html, acá puedes añadir una clase para el botón del `Header`, una clase como `className="header__button"`, y en scss hacer lo siguiente:
```suggestion
    &__button { ... }
```
Y así eliminar la clase `&__login {}`, mejorando la legibilidad y evitando el nesting.